compiler/src/dmd/initsem.d

```diff
@@ -707,6 +707,19 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, ref Typ
             return false;
         }
 
+        /* Run semantic on ExpInitializer, see if it represents entire struct ts
+         */
+        bool representsStruct(ExpInitializer ei, TypeStruct ts)
```

Comment:
This should be private.

Reply:
It's a nested function, which is always private.

Reply:
Ah, I didn't notice it was a nested function. Sorry.